### ISSUE TITLE
Ambient RNA correction in HTO samples - required?

### ISSUE THREAD

**Issue by learning-MD** [satijalab/seurat#6532]
From the original HTO paper (https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1603-1), the suggestion seems to be that by removing "Negative" barcodes, ambient RNA may be removed.

Is it recommended to apply an ambient RNA correction technique (e.g., CellBender, SoupX, DecontX, etc.) if you're working with hashed samples? Or, should I be reasonably confident that ambient RNA correction occurs by removing the Negative barcodes?

If ambient RNA correction is still recommended, would the preference be to run that only on barcodes filtered to include only "Singlets" or the entirety of the dataset before going through the HTO demultiplexing vignette?

Thanks.

**Comment by f6v** [comment]
I think you're confused by what "ambient RNA correction" does. The goal there is to estimate the contribution of background RNA to the UMI counts you observe for each barcode. The idea is that ambient(contaminating) RNA will be present in good-quality cells and distort the downstream results.

Now, whether HTO-negative barcodes are truly low-quality cells with high background content is something for you to figure out for your sample. They might as well be, or maybe the hashtagging wasn't optimal in the lab. As they mention in the paper, lower total UMI counts in HTO-negative barcodes could mean these are lower-quality cells.

If you simply remove these HTO-negative barcodes, that doesn't solve the ambient RNA problem(if there is one) for the good-quality cells. Whether you should try to adjust for background RNA depends on your sample. It's probably not going to work if you have only one homogeneous cell population in the data, like all your cells are T cells. 

**tl;dr** demultiplexing and ambient RNA correction are different things.

**Comment by mhkowalski** [comment]
Hi, I agree with f6v's comments. You may still apply ambient correction to hashed samples for exactly the reasons above. We have not assessed whether you should run ambient RNA correction before or after demultiplexing and this is not really in the scope of this Seurat. Many of these algorithms use empty droplets for inference of ambient RNA, so you would want to make sure to retain these empty droplets in either case. I could see an argument for removing doublets prior to ambient RNA correction but I would try this out yourself or consult the developer of your tool of choice.
